From 2fbf33172d39b5af4d1c9c171b075c153591ce27 Mon Sep 17 00:00:00 2001
From: Prasanna Balan <prazzb@gmail.com>
Date: Wed, 14 Aug 2013 08:20:44 +0530
Subject: [PATCH] remove superfluous is_system_libdir check

this caused -L/lib not being printed when prefix= was used.
the test was bogus anyway, because it used strcmp on non-reduced paths
directly. so //lib and /lib were 2 different things, when in reality
they are not.
---
 main.c | 21 ---------------------
 1 file changed, 21 deletions(-)

diff --git a/main.c b/main.c
index 618d49a..3efd2d0 100644
--- a/main.c
+++ b/main.c
@@ -56,30 +56,9 @@ static char *sysroot_dir = NULL;
 
 FILE *error_msgout = NULL;
 
-static bool
-fragment_has_system_dir(pkg_fragment_t *frag)
-{
-	switch (frag->type)
-	{
-	case 'L':
-		if ((want_flags & PKG_KEEP_SYSTEM_CFLAGS) == 0 && !strcasecmp(SYSTEM_LIBDIR, frag->data))
-			return true;
-	case 'I':
-		if ((want_flags & PKG_KEEP_SYSTEM_LIBS) == 0 && !strcasecmp(SYSTEM_INCLUDEDIR, frag->data))
-			return true;
-	default:
-		break;
-	}
-
-	return false;
-}
-
 static void
 print_fragment(pkg_fragment_t *frag)
 {
-	if (fragment_has_system_dir(frag))
-		return;
-
 	if (frag->type)
 		printf("-%c%s ", frag->type, frag->data);
 	else
-- 
1.8.1.5

